elizhahn/karaoke-stretch
# Feature/user messages

## What does this PR do (summary of changes)?
 - Adds user message for search results
 - Adds user message for status code errors
 
## How should it be tested?
 - Make a valid and invalid search to see search messages
 - Stop server to check status code error message
 
## Future Iterations:
 - We can mix up the language or styling of the messages if you want!
